target/linux/ar71xx/files/arch/mips/ath79/mach-rbspi.c

```diff
@@ -775,6 +777,10 @@ static void __init rb750upr2_setup(void)
 	/* differentiate the hEX lite from the hEX PoE lite */
 	if (strstr(mips_get_machine_name(), "750UP r2"))
 		flags |= RBSPI_HAS_USB | RBSPI_HAS_POE;
+	
+	/* differentiate the Powerbox from the hEX lite */
+	if (strstr(mips_get_machine_name(), "750P r2"))
```

Comment:
Just a nitpick but this should be an `else if ()`: both tests are mutually exclusive.

Reply:
Thanks for the catch.
Corrected it